modules/acquia_cms_headless/tests/src/Functional/HeadlessHybridModeTest.php

```diff
@@ -98,4 +151,13 @@ protected function installModule(string $module) {
     return $this->moduleInstaller->install([$module]);
   }
 
+  protected function childMenuTest($menu, $parent, $childs, $selector) {
```

Comment:
@rajeshreeputra add small functional description of this method.

Reply:
done